developers/blobstream.md

```diff
@@ -14,128 +16,154 @@ on Ethereum, for integration by developers into L2 contracts. This enables Ether
+
+Block ranges that are before the contract's `latestBlock` can't be
+proven a second time in different batches.
+
+More information can be found in [`requestHeaderRange(...)`](https://github.com/succinctlabs/blobstreamx/blob/364d3dc8c8dc9fd44b6f9f049cfb18479e56cec4/contracts/src/BlobstreamX.sol#L78-L101)
+method.
 :::
 
-## How Blobstream works
-
-Blobstream allows Celestia block header data roots to be relayed in one
-direction, from Celestia to an EVM chain. It does not support bridging
-assets such as fungible or non-fungible tokens directly, and cannot send
-messages from the EVM chain back to Celestia.
+### How Blobstream X works
 
-It works by relying on a set of signers to attest to some event on Celestia:
-the Celestia validator set. The Blobstream contract keeps track of the
-Celestia validator set by updating its view of the validator set with
-`updateValidatorSet()`. More than 2/3 of the voting power of the current
-view of the validator set must sign off on new relayed events, submitted with
-`submitDataRootTupleRoot()`. Each event is a batch of `DataRootTuple`s, with
-each tuple representing a single
-[data root (i.e. block header)](https://celestiaorg.github.io/celestia-app/specs/data_structures.html#header).
-Relayed tuples are in the same order as Celestia block headers.
+As shown in the diagram below, the entrypoint for updates to the Blobstream
+X contract is through the `SuccinctGateway` smart contract, which is a
+simple entrypoint contract that verifies proofs (against a deployed
+onchain verifier for the Blobstream X circuit) and then calls the
+`BlobstreamX.sol` contract to update it.
+[Find more information about the `SuccinctGateway`](https://docs.succinct.xyz/platform/onchain-integration#succinct-gateway).
 
-![Blobstream attestation flow](/img/blobstream/Celestia_Blobstream_attestation_flow.jpg)
+![blobstream x overview diagram draft](/img/blobstream/Celestia_Blobstream_X2b.png)
 
-### Events and messages relayed
+<!-- markdownlint-disable MD042 -->
 
-**Validator sets**:
-The relayer informs the Blobstream contract who are the current
-validators and their power.
-This results in an execution of the `updateValidatorSet` function.
-
-**Batches**:
-The relayer informs the Blobstream contract of new data root tuple roots.
-This results in an execution of the `submitDataRootTupleRoot` function.
+:::tip NOTE
+If the Blobstream X contract is not deployed on a desired chain,
+it needs to be deployed before it can be used by your rollup. See the
+[deployment documentation](https://docs.succinct.xyz/platform/onchain-integration#gateway-deployment)
+for more details.
+:::
 
-## How to integrate
+### How to integrate with Blobstream X
 
-Integrating your L2 with Blobstream requires two components: your onchain smart
+Integrating your L2 with Blobstream X requires two components: your onchain smart
 contract logic, and your offchain client logic. The next three sections cover these
```

Comment:
`what's the offchain client logic`? is it access to an RPC? if so, then I prefer stating that clearly

Reply:
it will be covered in here https://docs.celestia.org/developers/blobstream-offchain. should i change the path of the page back to `blobstream-offchain` instead of `blobstream-x-offchain` @rach-id ?

Reply:
and other pages: 
/developers/blobstream-contracts
/developers/blobstream-offchain
/developers/blobstream-proof-queries

Reply:
thanks 🙏

Reply:
https://github.com/celestiaorg/docs/pull/1379/commits/31360c21ad0b92e50f12d04396d28f471e0c1cdc